src/core/src/core_logic/PatchInstaller.py

```diff
@@ -177,14 +186,44 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
         self.last_still_needed_packages = all_packages
         self.last_still_needed_package_versions = all_package_versions
 
+        batch_patching_stopwatch = Stopwatch(self.env_layer, self.telemetry_writer, self.composite_logger)
+        batch_patching_stopwatch.start()
+
+        installed_update_count, patch_installation_successful, maintenance_window_batch_cutoff_reached = self.install_patches_in_batches(all_packages, packages, package_versions, maintenance_window, package_manager)
+
+        batch_patching_stopwatch.stop()
+
+        packages, package_versions = self.get_remaining_packages_to_install(package_manager) # These packages should be: (a) Those which were failed while installing in batches and (b) some new packages available in this brief time when batch patching was being done.
```

Comment:
Update the comment to add one more type of packages: the packages which could not be installed due to there was not enough remaining time to install the packages in batches.

Reply:
Updated